module-1/src/Components/MovieListPage/MovieListPage.tsx

```diff
@@ -1,179 +1,179 @@
-import { useEffect, useState } from "react";
-import { useSearchParams } from "react-router-dom";
-import { TestData } from "../../TestData";
-import { Genres } from "../Genres/Genres";
-import { MovieTile } from "../MovieTile/MovieTile";
-import { Search } from "../Search/Search";
-import { SortControl } from "../SortControl/SortControl";
-
-let sortValues = TestData.SearchFields.slice(0, 5);
-let listOfGenres = TestData.genres;
-const EMPTYSTRING = "";
-const MAINURL = "http://localhost:4000/movies?limit=20";
-
-export const MovieListPage = () => {
-  const [selectedGenre, setSelectedGenre] = useState(listOfGenres[0].genreName);
-  const [sortByField, setSortByField] = useState("title");
-  const [searchValue, setSearchValue] = useState("");
-  const [movies, setMovies] = useState<Movie[]>([]);
-  const [isLoading, setIsLoading] = useState(true);
-  const [searchparams, setSearchParams] = useSearchParams();
-
-  useEffect(() => {
-    fetchUrl(buildUrl());
-  }, []);
-
-  const buildUrl = (operation: string = "", value: string = "") => {
-    let newUrl = "";
-    let sort_str = "";
-    let search_str = "";
-    let genre_str = "";
-    let paramsObject = {};
-    switch (operation) {
-      case "":
-        sort_str = `&sortBy=${sortByField}&sortOrder=asc`;
-        newUrl = `${MAINURL}${sort_str}`;
-        paramsObject = { sortBy: sortByField, sortOrder: "asc" };
-        break;
-
-      case "genre":
-        sort_str = `&sortBy=${sortByField}&sortOrder=asc`;
-        search_str =
-          searchValue === EMPTYSTRING
-            ? EMPTYSTRING
-            : `&search=${searchValue}&searchBy=title`;
-        genre_str = value === "All" ? EMPTYSTRING : `&filter=${value}`;
-        newUrl = `${MAINURL}${sort_str}${search_str}${genre_str}`;
-        paramsObject = { sortBy: sortByField, sortOrder: "asc" };
-        if (search_str.length > 0) {
-          paramsObject = {
-            ...paramsObject,
-            search: searchValue,
-            searchBy: "title",
-          };
+  import { useEffect, useState } from "react";
```

Comment:
Something's wrong with the diff here and throughout the PR, I'm guessing it's CR/CRLF enter symbol change. Please check setting for your formatter and ensure only changed lines are in the diff.